### PR TITLE
Add: added home-page for spacTrend website

### PR DESCRIPTION
**Pull request by sakleprajwal**
Home-page is added in spacTrend website which is an E-commerce company of fashion & lifestyle products.

image

image

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    display: flex;
+    justify-content: center;
+}
+
+.search{
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    min-width: 3rem;
+    margin: 0 1rem 0 1rem;
```
Comment (Sameony) [inline css/homepage.css]: `margin: 0 1rem`
will act the same as this, so you can replace that

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    justify-content: space-evenly;
+    align-items: center;
+    color: var(--ternary-color);
+}
+
+.navbar-icon-btn{
+    border: none;
+    background-color: transparent;
+    font-size: 1.2rem;
+    padding: 0 8px 0 8px;
```
Comment (Sameony) [inline css/homepage.css]: you can use the shorthand 
`padding: 0 8px` for this

File: style.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    --brand-font: 'Lobster', cursive;
+    --primary-font: 'Playfair Display', serif; 
+    --secondary-font: 'Asap', sans-serif; 
+    --ternary-font: 'Lato', sans-serif;  
+    /* 
+    font-family: 'Lobster', cursive;            B
+    font-family: 'Playfair Display', serif;     Deal of the day
+    font-family: 'Asap', sans-serif;            BRAND/PRICE/COLOR
+    font-family: 'Lato', sans-serif;            Paragraph 300/400/700
+    */
```
Comment (Sameony) [inline style.css]: remove unnecessary blocks of code before merging

**Review by Sameony — APPROVED** [review]
Suggested Minor changes, do follow up before merging

File: css/homepage.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
+body{
+    margin: 0;
+    font-family: 'Lato', sans-serif;
+}
+
+.navbar-section{
+    display: flex;
+    flex-direction: row;
```
Comment (krishanu2911) [inline css/homepage.css]: no need to give flex-direction row it is by default it's in row

File: css/homepage.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    max-width: 20rem;
+    font-family: var(--brand-font);
+    font-size: 2rem;
+    display: flex;
+    justify-content: center;
+}
+
+.search{
+    display: flex;
+    flex-direction: row;
```
Comment (krishanu2911) [inline css/homepage.css]: same here

File: css/homepage.css [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+.navbar-actions{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 100%;
+    max-width: 20rem;
+}
+
+.categories-section{
+    display: flex;
```
Comment (krishanu2911) [inline css/homepage.css]: also like this property is repeating many times so you can move this to utils

File: index.html [hunk 7]
```diff
@@ -1 +1,63 @@
...
+    </div>
 
+    <div class="categories-section">
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category1</h4></div>
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category2</h4></div>
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category3</h4></div>
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category4</h4></div>
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category5</h4></div>
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category6</h4></div>
+        <div class="category-item"><img src="asset/fashion.png" alt="" srcset=""> <h4>Category7</h4></div>
```
Comment (krishanu2911) [inline index.html]: Add alt and srcset in the tag

**Review by krishanu2911 — APPROVED** [review]
Added few suggestion else everything is good to go

File: css/homepage.css [hunk 8]
```diff
@@ -0,0 +1,159 @@
+body{
+    margin: 0;
```
Comment (kotalwar-krishna) [inline css/homepage.css]: if you want to use margin zero for whole body by default put it in *{ }

File: css/homepage.css (hunk 2 above)
Comment (kotalwar-krishna) [inline css/homepage.css]: please consistent while using units. use rem saggestion

File: css/homepage.css [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+.search-btn{
+    background-color: var(--ternary-color);
+}
+
+.navbar-actions{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 100%;
+    max-width: 20rem;
```
Comment (kotalwar-krishna) [inline css/homepage.css:59]: if you have to use max- width so why width 100%. Remove max-width and cheack

File: css/homepage.css [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+    flex-direction: row;
+    justify-content: center;
+    overflow: scroll;
+}
+
+.category-item{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 8px 2rem 8px 2rem;
```
Comment (kotalwar-krishna) [inline css/homepage.css]: same saggestion use short-hand like padding: 8px 2rem; and please be consistent with units.

**Review by kotalwar-krishna — APPROVED** [review]
Code looks good. 

some saggestions: 1.be consistent with units. mostly use rem.
                               2. as you use shorthand for margin. use it for padding also.

File: style.css (hunk 3 above)
Comment (sakleprajwal) [reply style.css]: Actually, it is not unnecessary. I have kept it to remember when to use which font. Is it ok if I keep it?

File: css/homepage.css (hunk 6 above)
Comment (sakleprajwal) [reply css/homepage.css]: I have created a util class and it has significantly reduced the length of code. Thank you so much.

File: css/homepage.css (hunk 9 above)
Comment (sakleprajwal) [reply css/homepage.css:59]: With width: 100% & max-width: 20rem
image

With max-20rem
image

File: style.css (hunk 3 above)
Comment (Sameony) [reply style.css]: it might create misunderstandings in middle of the code
if you wish to keep it, do place it at the bottom/top-most

**Pull request merged**